examples/shopping-cart/components/Cart.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+var React = require('react');
+
+var Product = React.createClass({
+  render: function() {
+    return <div > {
+      this.props.children
+    } < /div>;
+  }
+});
+
+var Cart = React.createClass({
+  propTypes: {
+    products: React.PropTypes.arrayOf(React.PropTypes.shape({
+      id: React.PropTypes.number.isRequired,
+      title: React.PropTypes.string.isRequired,
+      price: React.PropTypes.number.isRequired,
+      quantity: React.PropTypes.number.isRequired,
+    })).isRequired,
+    total: React.PropTypes.string.isRequired,
+    onCheckoutClicked: React.PropTypes.func.isRequired
+  },
+
+  render: function() {
+    var products = this.props.products;
+
+    var hasProducts = products.length > 0;
+    var nodes = !hasProducts ?
+      < div > Please add some products to cart. < /div> :
+    products.map(function(p) {
+      return <Product key = {
+        p.id
+      } > {
+        p.title
+      } - & euro; {
+        p.price
+      }
+      x {
+        p.quantity
+      } < /Product>;
+    });
+
+    return ( < div className = "cart uk-panel uk-panel-box uk-panel-box-primary" >
+      < div className = "uk-badge uk-margin-bottom" > Your Cart < /div> < div className = "uk-margin-small-bottom" > {
```

Comment:
@npverni Weird side effect of the prettifying tool I used to convert from 4 space -> 2 space indentation.  Fixed!  Still need to work on the linting issues.